src/house.cpp

```diff
@@ -391,26 +391,24 @@ void AccessList::parseList(std::string_view list)
 		}
 
 		boost::algorithm::trim(line);
-
 		if (line.empty() || line.front() == '#' || line.length() > 100) {
 			continue;
 		}
 
-		boost::algorithm::to_lower(line);
-
 		std::string::size_type at_pos = line.find("@");
 		if (at_pos != std::string::npos) {
 			if (at_pos == 0) {
 				addGuild(line.substr(1));
 			} else {
-				addGuildRank(line.substr(0, at_pos - 1), line.substr(at_pos + 1));
+				addGuildRank(line.substr(0, at_pos), line.substr(at_pos + 1));
 			}
 		} else if (line == "*") {
 			allowEveryone = true;
 		} else if (line.find("!") != std::string::npos || line.find("*") != std::string::npos ||
 		           line.find("?") != std::string::npos) {
 			continue; // regexp no longer supported
 		} else {
+			boost::algorithm::to_lower(line);
```

Comment:
sorry to say, but this is also not needed
to_lower is already used in getPlayerByName here: https://github.com/otland/forgottenserver/blob/master/src/game.cpp#L438